content/source/docs/enterprise/private/updating-license.html.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: "enterprise2"
+page_title: "Updating License Key"
+sidebar_current: "docs-enterprise2-private-installer-rhel"
+---
+
+# Updating License Key
+
+This guide is for updating the License file for an installation of Private Terraform Enterprise.
+
+If you are missing the new License Key, please talk to to your **Solutions Engineer** (pre-sales, POC or trial) or **Technical Account Manager** (existing customers).
+
+## Requirements
+
+* shell access to the VM running Private Terraform Enterprise
+* The license file (with the extension `.rli`)
+
+## Steps
+
+### If using Replicated > `Version`
+
+1. Copy the new license file to the PTFE instance (for example with `scp`)
+2. `ssh` to the PTFE instance
+3. Load the license, by running `cat <license_file> | replicatedctl license-load`
+4. In the PTFE Admin Console, select the "View License" menu under the gear icon (or navigate directly to <host>:8800/license.
+5. Click the "Sync License" button.
+6. Go to the dashboard using the Dashboard menu.
+7. Restart the PTFE application in the dashboard.
+
+### If using Replicated < `Version`
+
+1. Copy the new license file to the PTFE instance (for example with `scp`)
+2. `ssh` to the PTFE instance
+3. Set the license file location with the command line: `replicatedctl params set LicenseFileLocation --value '/full/path/to/license.rli`
+4. Stop PTFE in the Web UI under `<host>:8800/`
+4. Stop Replicated:
```

Comment:
For these code blocks nested inside a list item:

- Add a blank line before and after the code block.
- Indent the entire code block 4 spaces, including the triple-backtick delimiters. 
- Use `shell` as the language, and prefix the lines with `$ `.

Reply:
Fixed 👍